### PR TITLE
feat: rebuild distribution pipeline for self contained styles

### PR DESCRIPTION
**Pull request by jbadan**
## Description
**[BREAKING CHANGE]**
### Changes
1. `components` renamed to `src` (this will now be the **ONLY** published folder). We are not publishing `scss` directory. Although, fonts, icons and images will be copied into `dist` folder until another solution is created.
1. set default path for icons and fonts to reduce need to setting variable in each scs component file.
1. in `src`:
   - (unfinished scs) layout components so they are published as one file
   - `helpers` file
   - created new `src/fundamental-styles.scss` master list for use via cdn or for docs sites. Removed old references.
   - `core/_root` now available as `src/root` for variable fallbacks for postcss config. 
1. removed `ie11-fallback`, no longer needed due to postcss config.
1. cleaned up mixins/functions. 
1. removed `scss/core/_elements` file which contained a bunch of base element styles that were affecting self contained styles.

#### Docs site
1. removed hero image on homepage as it is no longer relevant 
1. removed how-to-use-scss in angular page - no longer relevant 
1. added css that the docs site was dependent on from now deleted `scss/core/_elements` 
1. replaced search with `searchInput` pattern 

#### Playground
1. removed other playgrounds besides "new" default
1. default playground now serves css from `src` directory (self contained styles)
     - TO DO in a followup story: add postcss config to node-sass build step for playground so we can verify ie11 styles

#### Visual Regression Testing
1. fresh new look - reduced amount of "magic" happening
1. tests now reflect self contained styles, not `fundamental-styles.scss`
1. cleaned up naming convention 
    - `test/visual-regression-tests/resources/reference_images/visual_regression_fundamental_action-bar_0_example-containerfd-action-bar_0_desktop.png` --> 
    -  `test/visual-regression-tests/reference_images/FS_action-bar_0_backstop_0_.png`
1. screenshots will now be taken of each `format(example)` block on component pages. This simplifies and clarifies the process, as well as exposing that we were missing about 500 examples 👎 

**TO DO**
- update wiki pages

## Screenshots
There are way too many changes to screenshot.

### CODE REVIEW

**Comment by jbadan** [comment]
I'm not sure why the netlify preview is linked to the wrong pr here- but this is the correct link: https://5d712dd5dde5f00008a1b5d0--fundamental-styles.netlify.com/

**Comment by droshev** [comment]
the first search input example doesn't seem to work

File: README.md [hunk 1]
```diff
@@ -20,21 +20,20 @@ The library is modular so you can use as little or as much as you need.
...
-<link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.min.css" rel="stylesheet">
+<link href="//unpkg.com/fundamental-styles@latest/dist/fundamental-styles.css" rel="stylesheet">
 ```
 
 ### NPM Package
 
-The compiled CSS for the full library and modules, e.g., core, layout, etc., are distributed via [NPM](https://www.npmjs.com/package/fundamental-styles) along with the SASS source.
+The compiled CSS for the full library and modules, e.g., core, layout, etc., are distributed via [NPM](https://www.npmjs.com/package/fundamental-styles).
 
 This gives you incredible flexibility to use individual components and enable [advanced customization](https://github.com/SAP/fundamental-styles/wiki/Advanced-Customization) options.
```
Comment (bcullman) [inline README.md]: url does not exist, and perhap neither does the concept.

Comment (jbadan) [reply README.md]: Deleted.

Comment (bcullman) [reply README.md]: actually, i meant this line

> This gives you incredible flexibility to use individual components and enable [advanced customization](https://github.com/SAP/fundamental-styles/wiki/Advanced-Customization) options.

Comment (jbadan) [reply README.md]: Yeah sorry I meant deleted locally, I hadn't pushed it yet :)

**Pull request merged**